cmd/server/main.go

```diff
@@ -31,6 +31,8 @@ func main() {
 		panic(err)
 	}
 	s := server.NewServer(apiGroups...)
+
+	klog.Info("starting api server ...")
```

Comment:
Could you please add the printing of version as well here?

Reply:
fixed it